### PR TITLE
added translators folder and moved translators to base, added DeepL translations

### PR DESCRIPTION
**Pull request by ArnarTumi**

### CODE REVIEW

File: src/wagtail_translate/apps.py [hunk 1]
```diff
@@ -7,5 +7,56 @@ class WagtailTranslateAppConfig(AppConfig):
...
+            # Differentiate between regular Django model and Wagtail Page.
+            # - Page instances have `save_revision` and `publish` methods.
+            # - Regular Django model (aka Wagtail Snippet) need to be saved.
+            if isinstance(translated_obj, Page):
+                # Calling `publish` is optional,
+                # and will publish the translated page.
+                # Without, the page will be in draft mode.
+                translated_obj.save_revision().publish()
+            else:
+                translated_obj.save()
```
Comment (allcaps) [inline src/wagtail_translate/apps.py]: The intention is that this should live in your own app, and not in Wagtail Translate.

So, lets leave this file unchanged and keep the `from . import monkeypatches # noqa` (which we wrap in a wagtail version conditional at some point).

**Review by allcaps — CHANGES_REQUESTED** [review]
Just revert the apps.py changes. The rest is good to go. Thank you!

File: src/wagtail_translate/apps.py (hunk 1 above)
Comment (ArnarTumi) [reply src/wagtail_translate/apps.py]: ah, yes, sorry, will revert!

**Comment by ArnarTumi** [comment]
Done

**Review by allcaps — APPROVED** [review]

**Pull request merged**